python/Quickstart.MD

```diff
@@ -5,130 +5,107 @@ This quickstart covers:
-		"materialProblem": "SHORT_SHIPMENT_RECEIVED",
-		"isEscalated": false,
-		"createdByPartner": false,
-		"isSubmittedToPartner": false,
-		"directSupplierImpact": {
-			"businessImpact": "Business Impact",
-			"businessPriority": "HIGH"
-			}
-		}
+request_body = {
+  "header": {
+    "headerVersion": 1,
+    "eventName": "agile-process-teams:add-direct-supplier-incident:v3",
+    "ownerId": "YOUR_OWNER_ID",
+    "processNetworkId": "YOUR_PROCESS_NETWORK_ID",
+    "appName": "agile-process-teams",
+    "dataspace": "default"
+    },
+  "payload": {
+    "aptBusinessObjectSummary": "Incomplete shipment received.",
+    "aptBusinessObjectDescription": "Several packages missing or damaged.",
+    "deviationType": "UNPLANNED",
+    "materialType": "FINISHED_GOOD",
+    "materialSubtype": "SHIPPED",
+    "materialProblem": "SHORT_SHIPMENT_RECEIVED",
+    "isEscalated": False,
+    "createdByPartner": False,
+    "isSubmittedToPartner": False,
+    "directSupplierImpact": {
+      "businessImpact": "Business Impact",
+      "businessPriority": "HIGH"
+    }
+  }
 }
 ```
+This creates a dictionary object containing the `header` and `payload` elements we discussed earlier.  You can work with the items in this dictionary using standard Python methods.  This allows you to make any desired modifications prior to packaging and submitting your request.  
 
-The `utils.py` file has functions to help build the payload for your request body. The `read_payload_file(filename)` function reads the contents of the file, loads it into a JSON object, and does some minor processing for insertion into the request body.
+As an example, let's change the priority of this incident from `HIGH` to `MEDIUM`.  To change this value, we must reference the `businessPriority` item.  To reach this value, reference the `directSupplierImpact` element of the `payload`.  
 
-In your Python terminal, enter the following:  
+**Type or copy the following into your Python terminal**
 ```
->>> import utils
->>> my_file = 'addIncident.json'
->>> payload_data = utils.read_payload_file(my_file)
+request_body.get('payload').get('directSupplierImpact').update({'businessPriority': 'MEDIUM'})
 ```
-If you already use a JSON payload as part of your other processes, you can the `accept_json_payload(payload_in)` function instead. This function accepts a JSON object, checks for proper format, and does some minor processing for insertion into the request body.
+We use the `.get()` method for each element in our tree that we wish to access.  The `.update()` method allows us to change a value by referencing its key.
+Use care to include the braces `{}` when updating an item.  
 
-In the Python terminal, enter the following:
-```
->>> my_json_data = result_from_some_other_process()
->>> payload_data = utils.accept_json_payload(my_json_data)
+Take a moment to add your `ownerId` and `processNetworkId` values using the method outlined above.  Some sample code is below as a hint:  
```

Comment:
I think this should be the first step -- move up and include with my earlier feedback about wording

Reply:
I want to leave this here.  If we establish the variables before the payload, the only advantage we get is that we can build the whole request dynamically.  What we lose is the opportunity to introduce the update method before we get too deep into the instruction.  This update method will be very important for most of the automation process.